uk.co.compucorp.civicrm.civihrcore/CRM/CiviHRCore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CRM_CiviHRCore {
+  /**
+   * Get version value from info.xml. This method uses caching so it reads
+   * xml file only once during CiviCRM session.
+   *
+   * @return string
+   */
+  public static function getVersion() {
```

Comment:
@davialexandre didn't you say we should avoid static methods if possible? Although this really do seem something that just needs to be static, but worth bringing this up

Is it possible to write a test for this?